web-animations-1/Overview.bs

```diff
@@ -708,6 +701,32 @@ animation model to change, but these operations in themselves do not create
 a new [=animation frame=], rather they merely update the current <a>animation
 frame</a>.
 
+### Timeline Phase ### {#timeline-phases}
+
+A [=timeline=] may be in one of four possible 
+<dfn export lt="timeline phase">phases</dfn> for each of which, a non-normative
+description is also provided:
+
+<div class=informative-bg>
+
+:   <dfn export lt="timeline inactive phase">inactive</dfn>
+::  A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive
+    </dfn> when its <a>time value</a> is <a>unresolved</a>.
+:   <dfn export lt="timeline before phase">before</dfn>
+::  The timeline current time is before its start point.
+:   <dfn export lt="timeline active phase">active</dfn>
+::  The timeline current time is after its start point and before its end point.
+:   <dfn export lt="timeline after phase">after</dfn>
+::  The timeline current time is at or beyond its end point.
+
+</div>
+
+[=timeline phase|Timeline phase=] is used to give additional context to 
+[=timeline current time|current time=]. For example, if a [=timeline=] has a
+delayed start, [=timeline current time|current time=] of 0 would be ambiguous
```

Comment:
What is a delayed start?

Reply:
Added example: "a ScrollTimeline startScrollOffset greater than 0"